.github/workflows/slowtest.yaml

```diff
@@ -0,0 +1,64 @@
+name: SlowEst
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+    tags:
+      # YYYYMMDD
+      - "20[0-9][0-9][0-1][0-9][0-3][0-9]*"
+  schedule:
+    - cron: "0 0 * * 1"
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  RUSTFLAGS: '--cfg async_executor_impl="async-std" --cfg async_channel_impl="async-std"'
+  RUST_LOG: info,libp2p=off,node=error
+
+jobs:
+  slowest:
```

Comment:
Or is "slowest" chosen on purpose?

Reply:
Yes. I was it was a play on `nextest` plus these are in fact the slowest tests.